docs/content/direct/architecture.md

```diff
@@ -77,13 +82,37 @@ More details on the internals of this module are provided in [KubeStellar Contro
 
 ## Pluggable Transport Controller
 
-* The pluggable transport controller watches `Binding` objects on the WDS, and maintains in the Inventory and Transport Space (ITS) 
-a wrapped object per `Binding` to be delivered.
-* This controller is pluggable and can potentially be implemented using different options. Currently the only option we support is based on the [Open Cluster Management Project](https://open-cluster-management.io) 
-
-There is one instance of the pluggable transport controller for each WDS. 
-Currently this controller runs in an executable process. This is a work in progress and we're working on running this controller in a dedicated pod.
-More details on the internals of this module are provided in [KubeStellar Controllers Architecture](#kubestellar-controllers-architecture).
+This controller's job is to (a) get workload objects from WDS to WECs
+as prescribed by the `Binding` objects and their referenced
+`CustomTransform` objects and inventory objects and (b) get
```

Comment:
Is (b) done by the OCM status addon?

Reply:
This paragraph is stating the responsibilities of the chosen transport; how the one based on OCM gets these done comes later.

Reply:
There is a bigger conceptual/terminological muddle that already existed and I am _not_ trying to untangle in this PR. I would say that KubeStellar has "pluggable transport", and the one implementation is based on OCM and involves several things: the ocm-transport-controller Deployment/Pod, the use of OCM, and the extension of OCM with the OCM Status Add-On (which, like every OCM extension, has both a "Controller" and an "Agent").

Reply:
I have added a few more words, trying to be a little clearer without taking on a major rewriting of the whole topic.